src/components/PluginSidebar.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div class="single-plugin-aside bg-gray-100 space-y-3">
+  <div class="single-plugin-aside bg-gray-100 space-y-3 min-w-fit order-first md:order-last">
```

Comment:
@rwfeather what do you think about adding a little padding on the right for larger screens and a little padding on the left for smaller screens? 

On larger screens I think the code blocks in particular would look better with some padding on the right:
![Screenshot 2022-09-23 at 3 51 47 PM](https://user-images.githubusercontent.com/9891457/192054710-a390c5ab-adba-42fa-8be0-02c125df4010.png)

And on smaller screens, I think it would look a little cleaner if the text lined up closer to the text of the main section: 
![Screenshot 2022-09-23 at 3 52 10 PM](https://user-images.githubusercontent.com/9891457/192054873-ec3ed6ce-6d0b-4112-a093-727a84c88ddf.png)

Reply:
related to this, I'm seeing some weird overflow - like text wrapping isn't quite working.
![Screen Shot 2022-09-23 at 4 20 19 PM](https://user-images.githubusercontent.com/5394188/192058299-17568852-3646-415f-b0b2-7ec91da61a87.png)

Reply:
> related to this, I'm seeing some weird overflow - like text wrapping isn't quite working

@tayloramurphy quick update, I've been trying to resolve this issue today and it's proving to be tricky. getting this PR merged is my current focus

Reply:
Overflow issue resolved 👍